example/src/App.jsx

```diff
@@ -19,9 +19,8 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 
 // Firebase.
-import firebase from 'firebase/app';
-import 'firebase/auth';
-import StyledFirebaseAuth from 'react-firebaseui/StyledFirebaseAuth';
+import firebase from 'firebase/compat/app';
```

Comment:
Include the side effect import too, `import 'firebase/compat/auth';`

Reply:
Fix: 
https://github.com/firebase/firebaseui-web-react/blob/e9cf37bf5a8b621d92399ed4ca39185ecec65a50/example/src/App.jsx#L23